homeassistant/components/atome/sensor.py

```diff
@@ -129,30 +129,20 @@ def _retrieve_live(self):
                 "Live Data : Missing last value in values: %s: %s", values, error
             )
             error_during_retrieve = True
-        return error_during_retrieve
+        return not error_during_retrieve
 
     @Throttle(LIVE_SCAN_INTERVAL)
     def update_live_usage(self):
         """Return current power value."""
-        error_in_retrieve = self._retrieve_live()
+        retrieve_has_been_sucessful = self._retrieve_live()
```

Comment:
I think `ok` or `success` could be a more succinct name.

Reply:
Hello,
I will put `retrieve_success`